azure-iot-device/azure/iot/device/iothub/models/message.py

```diff
@@ -43,7 +43,9 @@ def __init__(
         self.user_id = None
         self.content_encoding = content_encoding
         self.content_type = content_type
+        # Open Question: could input_name and output_name fields be combined somehow (destination?)
```

Comment:
># Open Question: could input_name and output_name fields be combined somehow (destination?) [](start = 8, length = 91)

personal opinion: I don't see an advantage and you'd actually remove some of the meaning if you did that.